lib/tasks/evm_test_helper.rb

```diff
@@ -1,5 +1,6 @@
 module EvmTestHelper
-  VMDB_SPECS        = FileList["spec/**/*_spec.rb"].exclude(/^spec\/(replication|migrations|automation)/)
+  #TODO: Reuse the array of excluded directories for building the FileList exclusion
+  VMDB_EXCLUDED_SPEC_DIRECTORIES = %w(replication migrations automation).freeze
```

Comment:
Ah that stinks...would be nice if RSpec had a dry-run-list-files command of some sort so you could ask it